linux.gypi

```diff
@@ -8,8 +8,10 @@
         'NN_USE_IFADDRS',
         'NN_HAVE_MSG_CONTROL',
     ],
+    'cflags': [ '-g','-O3', '-Wall', '-Wextra', '-Wno-sign-compare', '-Wno-unused',
```

Comment:
the `-g` flag is not necessary.  Especially since optimizations are enabled, having debug symbols for things functions that were inlined makes debugging difficult.  Usually, you use `-g` with optimizations turned off.  We want optimizations, but not debug symbols.